conf/routes

```diff
@@ -65,16 +65,13 @@ PUT     /adminapi/setRole                                    @controllers.AdminC
 
 # Auditing tasks
 GET     /audit                                               @controllers.AuditController.audit(nextRegion: Option[String] ?= None)
-#GET     /audit/easyRegion                                    @controllers.AuditController.auditNewEasyRegion
 GET     /audit/region/:id                                    @controllers.AuditController.auditRegion(id: Int)
 GET     /audit/street/:id                                    @controllers.AuditController.auditStreet(id: Int)
 GET     /audit/recalculateStreetPriority                     @controllers.AuditPriorityController.recalculateStreetPriority
-GET     /audit/getRegionStreetPriority/:regionId             @controllers.AuditPriorityController.getRegionStreetPriority(regionId: Int)
 POST    /audit/comment                                       @controllers.AuditController.postComment
 POST    /audit/nostreetview                                  @controllers.AuditController.postNoStreetView
 
 # Task API.
-GET     /task                                                @controllers.TaskController.getTask
```

Comment:
I commented about this in TaskController, I'll copy it here: 
>This wasn't being used anywhere in the code, and I don't see how it would have been useful for testing purposes either, which is why I deleted it. Let me know if you use it for testing ever, or otherwise think we should keep it.